letsencrypt.lua

```diff
@@ -109,21 +109,19 @@ end
 local b64url = require'b64url'.encode
 package.preload['acme.error'] = function()
     return {
-        parse = function()
```

Comment:
This code is part of acme.error package from zash. I'd rather leave it intact so it can be easily updated in the future.